lib/monitor.js

```diff
@@ -40,14 +41,28 @@ Monitor.prototype.createMonitor = function (opt) {
 };
 
 /**
+ * Utility function for converting object into array of tags in the
+ * ['key1:value1', 'key2:value2'] format
+ */
+ Monitor.prototype._toTags = function (json) {
+  return Object.keys(json).map(function(key) {
+    return key + ':' + json[key];
+  });
+};
+
```

Comment:
how is that going to break code? the `_toTags` function _is_ making the assumption that `json[key]` is a string-compatible value as well... maybe making sure both things are strings and appending them is enough. you'd have to test with datadog to see if sending `one:two:three` would break them for some weird reason